spec/reservation_manager_spec.rb

```diff
@@ -0,0 +1,199 @@
+require_relative "spec_helper"
+
+require "awesome_print"
+require "date"
+
+describe "manager" do
+  let (:manager) {
+    Hotel::ReservationManager.new
+  }
+  let (:block1) {
+    manager.make_block(5, "march 19, 2019", "march 23, 2019", 1)
+  }
+  let (:reservation1) {
+    manager.make_reservation("march 15, 2019", "march 20, 2019")
+  }
+  let (:reservation2) {
+    manager.make_reservation("march 17, 2019", "march 22, 2019")
+  }
+  let (:block2) {
+    manager.make_block(4, "march 21, 2019", "march 25, 2019", 2)
+  }
+
+  describe "Reservation_manager instantiation" do
+    it "is an instance of Reservation Manager" do
+      manager
```

Comment:
Right now, the way that `let` variables work is... Within a test, if the test needs to grab `manager`, it will go to the let block to figure out what that variable is.

For example... if you commented/deleted the line above that just says `manager`... the line below still works! Because when your program executes the line below, it will go "What's a `manager`? Oh! Let me go to the `let` block."